app/templates/main/index.html

```diff
@@ -8,6 +8,12 @@
     <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
       <h1 class="tna-heading-xl">TNA Django application</h1>
       <p class="tna-large-paragraph">Get started building a National Archives Django application.</p>
+      <h2 class="tna-heading-l">Admin site</h2>
+      <ul class="tna-ul">
+        <li>
+          <a href="/admin/">Admin site</a>
```

Comment:
Also could display to `request.user.is_staff` users only.

Reply:
Is this site actually going to be used by users who are not staff?

Reply:
Probably not. It actually may be better for now people without permissions know about existence of this link.